go.mod

```diff
@@ -39,6 +39,8 @@ require (
 	github.com/go-logr/stdr v1.2.2 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
+	github.com/hashicorp/go-retryablehttp v0.7.1 // indirect
```

Comment:
@miparnisari :) . Looks like we already indirectly depend on this. Does that change your approach to #3 ?

Reply:
Nah, I used that in a test. It is either that or sleep for a second while the service comes up. But I hear it may not be FOSSA...